src/supremm/ingest_jobscripts.py

```diff
@@ -46,8 +47,16 @@ def __init__(self, dwconfig, schema):
                             `modw`.`job_tasks`
                         WHERE
                             resource_id = %s 
-                            AND local_job_id_raw = %s
-                            AND DATE(FROM_UNIXTIME(start_time_ts)) = %s"""
+                            AND local_job_id_raw = %s"""
+
+            if self.timestamp_mode == 'start':
+                self.query += ' AND ABS(DATEDIFF(DATE(FROM_UNIXTIME(start_time_ts)), %s)) < 2'
+            elif self.timestamp_mode == 'submit':
+                self.query += ' AND ABS(DATEDIFF(DATE(FROM_UNIXTIME(submit_time_ts)), %s)) < 2'
+            elif self.timestamp_mode == 'end':
+                self.query += ' AND ABS(DATEDIFF(DATE(FROM_UNIXTIME(end_time_ts)), %s)) < 2'
+            #else
+                # self.timestamp_mode == 'None' which means no date restrictions
```

Comment:
Remove commented-out code.

Reply:
this is not actually commented out code. It is a comment that describes the default case (since switch statements are not in python).

Reply:
Why not just write a comment instead of something indented like code?  It's also a bit confusing this elsewhere 'none' (lowercase) is used, but this uses 'None' which looks like the python `None`.  If this is just the way that you prefer to document these things then let me know.  It *looks* like code which is why I don't like it.